### PR TITLE
fixed issue #3400 and #3399

### PR DESCRIPTION
**Pull request by asif-webkul**
1. fixed issue #3400 UI for the compare page in default theme.
2.fixed issue #3399 Remove from wishlist button is displaying as 1 when mouse not hover on product, this occur in all locale except English.

### CODE REVIEW

File: packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
         }
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php:28]: Remove internal css

Comment (asif-webkul) [reply packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php]: done

Comment (shubhwebkul) [reply packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php]: execute **npm run prod** command in shop package

Comment (asif-webkul) [reply packages/Webkul/Shop/src/Resources/views/guest/compare/compare-products.blade.php]: excuted the command and pushed the code

**Pull request merged**